pages/reference/base-images/base-images.md

```diff
@@ -10,67 +10,66 @@ excerpt: Docker images maintained by {{ $names.company.lower }}
 ## Features Overview
 
 - Multiple Architectures:
-	- armv5e
-	- armv6
-	- armv7hf
-	- aarch64
-	- amd64
-	- i386
+  - armv5e
+  - armv6
+  - armv7hf
+  - aarch64
+  - amd64
+  - i386
 - Multiple Distributions:
-	- [Debian](https://www.debian.org/): jessie, sid, stretch and buster
-	- [Alpine](https://alpinelinux.org/): 3.6, 3.7, 3.8, 3.9, 3.10 and edge
-	- [Ubuntu](https://www.ubuntu.com/): xenial (16.04), bionic (18.04), cosmic (18.10), disco (19.04) and eoan (19.10)
-	- [Fedora](https://getfedora.org/): 26, 28, 29, 30 and 31
+  - [Debian](https://www.debian.org/): jessie, stretch, buster and sid
```

Comment:
I wonder if it's worth putting the Debian version numbers in brackets, similar to Ubuntu below?